tests/collections/asr/utils/test_audio_utils.py

```diff
@@ -39,6 +41,150 @@
 )
 
 
+class TestAudioSegment:
+    @pytest.mark.unit
+    @pytest.mark.parametrize(
+        "num_channels, channel_selectors", [(1, [None, 'average', 0]), (3, [None, 'average', 0, 1, [0, 1]]),]
```

Comment:
Can you add a few tests - 
- File path longer than 255 chars 
- filename longer than 255 chars
- two audio files incorrectly passed as different channel (raise appropriate error)

Reply:
Ok that seems like a reasonable error for (3). Let's add (1-2) then merge if they pass

Reply:
Perfect thanks !